src/Consoles/Screen.cs

```diff
@@ -68,7 +68,17 @@ public Screen(int width, int height)
                             new TestItem(AllowedItem.Single), new TestItem(AllowedItem.Single)
                         });
                     MenuConsole.OpenChest(chest);
-                };
+                };
+
+            // Nie wiem czy nie mamy buga bo w ten spos�b nie wy?wietlaj? si? najpierw poprzednie wiadomo?ci
+            // chyba ?e zawsze b?dziemy u?ywa? PrintMessageAndWait ale to podczas walki byloby slabe

```

Comment:
Opisz dokładniej jaką masz sytuację tu w komentarzu i unikajmy takich komentarzy w kodzie, bo go to w tym miejscu to nie dotyczy.